### ISSUE TITLE
Organization notifications UI changes

### ISSUE THREAD

**Issue by xavijam** [CartoDB/cartodb#11676]
A new feature for organization admins. The possibility to send notifications to the rest of organization users!. A preview:

![group](https://cloud.githubusercontent.com/assets/132146/23479826/027783c8-fec6-11e6-81a5-1c3e0570a3eb.png)

There are several small tasks in the frontend part (backend ticket), but it is not worth it to create several ones, so task are:

- [x] Create notifications page within organization admin.
- [x] Create notification description component (it doesn't mean it has to be javascript, in fact I'd go for a HTML form).
- [x] Create notification list and item (as I've explained above, I'd go for an easy one, html items with simple forms, for the action buttons).
- [ ] Check and make appear the new notifications appear under the notifications component in the header. https://github.com/CartoDB/cartodb/issues/11807
- [ ] When organization users visit her/his notifications, we should send a request setting that the notification has been read. https://github.com/CartoDB/cartodb/issues/11807

Design ticket: https://github.com/CartoDB/design/issues/582

**Comment by juanignaciosl** [comment]
Backend issues:

- [ ] Organization notification API #11677.
- [ ] User notification read API #11678.
- [ ] Mail notifications #11679.
- [ ] Mark notification as read API #11680.
- [ ] Automatic notification on new organization user #11681.

**Comment by asimcox** [comment]
@xavijam and @juanignaciosl -- I've updated the referenced ticket ^^ right above^^  to include all of the business and functional requirements we spoke about on Friday. It should be more clear now, but please let me know if you have any questions.

**Comment by asimcox** [comment]
@xavijam 
Designs have officially be approved for Manually Generated Notifications :) 
Hopefully that unblocks you!  

cc\ @noguerol @jorgesancha

**Comment by xavijam** [comment]
cc @matallo

**Comment by matallo** [comment]
currently working on

> Check and make appear the new notifications appear under the notifications component in the header.

> When organization users visit her/his notifications, we should send a request setting that the notification has been read.

I'm creating a separate issue for each one, so they can be more easyly approachable

**Comment by noguerol** [comment]
On my last check in staging layout has been broken, at least in Chrome 

![image](https://cloud.githubusercontent.com/assets/70603/24442837/370ca474-1460-11e7-9a05-4b2ef0b6bcb4.png)

**Comment by matallo** [comment]
yeah that seems totally off, I redeployed to staging again for acceptance, thanks @noguerol

**Comment by xavijam** [comment]
Deployed
